script/configure/update-frameworks

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -e
+
+mkframework() {
+  local prebuiltdir=$1
+  local frameworkdir=$2
+  local headersdir=$3
+  rm -rf $frameworkdir
+  install -d $frameworkdir
+  local framework="$(basename $frameworkdir | sed 's|\.framework$||g')"
+  echo "* Creating $frameworkdir/$framework"
+  for archdir in $prebuiltdir/{arm64,armv7s,i386,x86_64}; do
```

Comment:
Maybe parameterize these as they are used more times? (just a suggestion)

Reply:
Yeah, good point, doing that!

Reply:
Done in 1af850f0f6bda33908979574e2fc8e104ba86561